app/javascript/components/lease_ups/LeaseUpApplicationsFilterContainer.js

```diff
@@ -1,13 +1,16 @@
 import React, { useState } from 'react'
 
 import classNames from 'classnames'
+import { PropTypes } from 'prop-types'
```

Comment:
this should be `import PropTypes` not `import { PropTypes }`

```suggestion
import PropTypes from 'prop-types'
```